### ISSUE TITLE
hard to converge on Youtube-VIS

### ISSUE THREAD

**Issue by qqzy** [pixeli99/TrackDiffusion#18]
I am currently training on YouTube-VIS with a batch size of 6 (6 GPUs, 1 batch per GPU) and noticed that the model still hasn’t converged after 70K steps (Stage 1). May I ask how many iterations you trained for, and if there were any additional training strategies used? Thanks!

**Comment by pixeli99** [comment]
Hi qqzy,

For the training of diffusion, there may not be a significant decrease in loss, you may need to check the generated videos instead. Generally, 50K steps are already sufficient.

**Comment by pixeli99** [comment]
In addition, I'm not sure if what you mean by "not converging" refers to the loss or if there has been no bbox control effect？：）

**Comment by qqzy** [comment]
in my experiment, i find that there has been no bbox control effect even in training samples

**Comment by qqzy** [comment]
could you please tell the batch size setting in your experiment?

**Comment by pixeli99** [comment]
```
accelerate launch train_svd.py \
    --pretrained_model_name_or_path=/path/to/weight \
    --per_gpu_batch_size=1 --gradient_accumulation_steps=1 \
    --max_train_steps=50000 \
    --width=512 \
    --height=320 \
    --checkpointing_steps=1000 --checkpoints_total_limit=1 \
    --learning_rate=1e-5 --lr_warmup_steps=0 \
    --seed=123 \
    --mixed_precision="fp16" \
    --validation_steps=200
```
I used 8 A800 GPUs, which means my global bs is 8.
